### PR TITLE
added modal component

### PR DESCRIPTION
**Pull request by Anish-JS**
Please review modal component

### CODE REVIEW

File: components/Modal/Modal.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (shraddha15-creator) [inline components/Modal/Modal.html]: Change the Title

File: components/Modal/Modal.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        <p class="xs-padding para-sm-xs">
+          Lorem Ipsum is simply dummy text of the printing and typesetting
+          industry. Lorem Ipsum has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a galley of type and
+          scrambled it to make a type specimen book. It has survived not only
+          five centuries, but also the leap into electronic typesetting,
+          remaining essentially unchanged. It was popularised in the 1960s with
+          the release of Letraset sheets containing Lorem Ipsum passages, and
+          more recently with desktop publishing software like Aldus PageMaker
+          including versions of Lorem Ipsum.
```
Comment (shraddha15-creator) [inline components/Modal/Modal.html:29]: You can add some real content so it will look good to user

**Review by shraddha15-creator — CHANGES_REQUESTED** [review]

**Review by manismk — APPROVED** [review]

**Comment by Pratik1005** [comment]
Looks good, just add cursor pointer to open modal button

File: components/Modal/Modal.html (hunk 1 above)
Comment (Anish-JS) [reply components/Modal/Modal.html]: changed it

File: components/Modal/Modal.html (hunk 2 above)
Comment (Anish-JS) [reply components/Modal/Modal.html:29]: ok sure

**Comment by Anish-JS** [comment]
> Looks good, just add cursor pointer to open modal button

changed it

**Review by Pratik1005 — APPROVED** [review]

**Pull request merged**